setup/fs

```diff
@@ -116,9 +119,11 @@ mount_fs() {
 	for dev in $bdevs
 	do
 		local mnt=/fs/$(basename $dev)
-		log_cmd mkdir -p $mnt
+		[[ $(df | grep $dev | awk '{print $6}') == '/' ]] || log_cmd mkdir -p $mnt
```

Comment:
wonder when this will occur, you mean user choose to run the test on its root disc?

And help wrap this check in a function, and this logic need be in a separated commit, and in change log, try to describe the use scenario (including the lkp cmd) that the patch wants to support.

Reply:
this checks if the test is run on a local partition, checking if the disk is mounted on root